src/DropdownMenu.tsx

```diff
@@ -125,11 +116,13 @@ export function useDropdownMenu(options: UseDropdownMenuOptions = {}) {
       ...childArgs,
       props: {
         ...menuProps,
-        style: popper.styles as any,
+        ...(attributes.popper || {}),
```

Comment:
**nitpick:** you shouldn't need `|| {}` here? at least `|| null` or whatever should be enough. object spreads work with null-y values. actually, it works with any falsy value, so i'm not sure why we need a fallback here at all?